src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_up_commands.py

```diff
@@ -0,0 +1,296 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-few-public-methods
+
+import json
+import unittest
+import jmespath
+import tempfile
+import os
+
+from azure.cli.testsdk import (
+    ScenarioTest, ResourceGroupPreparer, JMESPathCheck, live_only)
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class WebAppUpE2ETests(ScenarioTest):
+    @live_only()
```

Comment:
@panchagnula may I know that why you make all your test live_only()? In this way, CI system still will skip the test.